dispatches/models/fossil_case/properties/hitecsalt_properties.py

```diff
@@ -96,32 +95,32 @@ def _make_params(self):
 
 #        Specific heat capacity at constant pressure (cp) coefficients
 #        Cp in J/kg/K
-        cp_param = {1: 5806,
-                    2: -10.833,
-                    3: 7.2413E-3}
+        cp_param = {1: 5806*pyunits.J/(pyunits.kg*pyunits.K),
```

Comment:
This is not the correct way to set units for a parameter. You should use the format `PAram(initialize=value, units=units)`